_posts/2024-05-28-pyos-newsletter-june-2024.md

```diff
@@ -0,0 +1,112 @@
+    </a>
+</figure>
+
+## <i class="fa-solid fa-handshake-simple"></i> See you at SciPy
+[SciPy 2024](https://www.scipy2024.scipy.org/) is just around the corner, and we can't wait to see you there! We're pulling together our pyOpenSci Guide to SciPy, similar to the guide we did for [PyCon 2024](https://www.pyopensci.org/blog/pyos-guide-to-pyconus-2024.html), and wanted to give you a preview of some of the tutorials and talks being given by pyOpenSci Community members!
+
+### Tutorials
+#### [3D Visualization with PyVista](https://cfp.scipy.org/2024/talk/GKGRWE/)
+Led by [Tetsuo Koyama](https://cfp.scipy.org/2024/speaker/WY7NA9/), [Alexander Kaszynski](https://cfp.scipy.org/2024/speaker/3RXHG8/), [Bill Little](https://cfp.scipy.org/2024/speaker/FDVWDN/), and [Bane Sullivan](https://cfp.scipy.org/2024/speaker/NEC33M/), this tutorial demonstrates [PyVista's](https://github.com/pyvista/pyvista) latest capabilities and bring a wide range of users to the forefront of 3D visualization in Python, including:
+
+* Use PyVista to create 3D visualizations from a variety of datasets in common formats.
+* Overview the classes and data structures of PyVista with real-world examples.
+* Be familiar of the various filters and features of PyVista.
+* Know which Python libraries are used and can be used by PyVista (meshio, trimesh etc).
+* We see this tutorial catering to anyone who wants to visualize data in any domain, and this ranges from basic Python users to advanced power users.
+
+#### [Create Your First Python Package: Make Your Python Code Easier to Share and Use](https://cfp.scipy.org/2024/talk/QT9GBY/)
+Led by pyOpenSci's Executive Director and Founder, [Leah Wasser](https://github.com/lwasser), in this tutorial session you'll learn how to:
+
+* Create code that can be installed into different environments
+* Use [Hatch](https://hatch.pypa.io/latest/) as a workflow tool, making setup and installation of your code easier
+* Use Hatch to publish your package to (test) PyPI
+
+### Talks
+#### [The power of community in solving scientific Python’s most challenging problems](https://cfp.scipy.org/2024/talk/AMTLJ7/)
+This talk from our Executive Director and Founder, [Leah Wasser](https://github.com/lwasser), shares pyOpenSci’s knowledge in building a constructive open peer review process that supports maintainers, and also resources that make Python packaging more accessible and easier-to-navigate. Our insights are based on five years of experience working with over 200 community members, who have evaluated 58 packages developed by over 56 package maintainers.
```

Comment:
you can link to our packaging guide here. this is about writing the guide via community.